controllers/tests/controller_server/utils_test.py

```diff
@@ -491,46 +489,47 @@ def _test_validate_parameters_match_volume(self, volume_field, volume_value, par
 
     def test_validate_parameters_match_volume_se_fail(self):
         with self.assertRaises(ValidationException):
-            self._test_validate_parameters_match_volume(volume_field="space_efficiency",
+            self._test_validate_parameters_match_volume(volume_field="space_efficiency_aliases",
                                                         volume_value=array_config.SPACE_EFFICIENCY_NONE,
                                                         parameter_field=controller_config.PARAMETERS_SPACE_EFFICIENCY,
                                                         parameter_value="thin")
 
     def test_validate_parameters_match_volume_thin_se_success(self):
-        self._test_validate_parameters_match_volume(volume_field="space_efficiency",
+        self._test_validate_parameters_match_volume(volume_field="space_efficiency_aliases",
                                                     volume_value=array_config.SPACE_EFFICIENCY_THIN,
                                                     parameter_field=controller_config.PARAMETERS_SPACE_EFFICIENCY,
                                                     parameter_value="thin")
 
+    def test_validate_parameters_match_volume_dedup_aliases_success(self):
+        self._test_validate_parameters_match_volume(volume_field="space_efficiency_aliases",
+                                                    volume_value=[array_config.SPACE_EFFICIENCY_DEDUPLICATED_COMPRESSED,
+                                                                  array_config.SPACE_EFFICIENCY_DEDUPLICATED],
+                                                    parameter_field=controller_config.PARAMETERS_SPACE_EFFICIENCY,
+                                                    parameter_value="deduplicated")
```

Comment:
not for this PR, but it would be shorter (and more generic), if `_test_validate_parameters_match_volume` would get only 2 parameters (instead of 4) of type `dict`: `volume_attributes` & `parameters`

Reply:
CSI-4597